src/index.html

```diff
@@ -11,7 +11,127 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="inblock">
+        <legend>Personal information</legend>
+        <label for="sName" class="inrow">Surname:
+          <input
+        type="text"
+        id="sName"
+        name="sName"
+        required
+        >
+        </label>
+        <label for="name" class="inrow">Name:
+          <input
+        type="text"
+        id="name"
+        name="name"
+        required
+        >
+        </label>
+        <label for="old" class="inrow">How old are You?
+          <input
+        type="number"
+        value="12"
+        name="old"
+        size="3"
+        min="1"
+        max="100"
+        >
+        </label>
+        <label for="birth" class="inrow">Full date of birth:
+          <input
+         type="date"
+         id="birth"
+         name="birth"
+         >
+        </label>
+        <label for="agree" class="inrow">I accept the term of the agreement
+          <input
+         type="checkbox"
+         id="agree"
+         name="agree"
+         >
+        </label>
+      </fieldset>
+      <fieldset class="inblock">
+        <legend>Registration</legend>
+        <label for="email" class="inrow">E-mail:
+          <input
+         type="email"
+         name="email"
+         placeholder="email@example.com"
+         >
+        </label>
+        <label for="pass" class="inrow">Password:
+          <input
+        type="password"
+        id="pass"
+        name="pass"
+        min="4"
+        max="10"
+        >
+        </label>
+      </fieldset>
+      <fieldset class="inblock">
+        <legend>An interesting fact about you!</legend>
+        <label for="cat" class="inrow">Do you love cats:
+          <input
+        type="radio"
+        name="cat"
+        > Yes
+          <input
+        type="radio"
+        name="cat"
+        > No
+        </label>
+        <label for="color" class="inrow">What is your favorite color:
+          <input
+        type="color"
+        id="color"
+        name="color"
+        >
+        </label>
+        <label for="timebad" class="inrow">What time do you go to bad?
+          <input
+        type="time"
+        id="timebad"
+        name="timebad">
+        </label>
+        <label for="cars" class="inrow">What are your favorite brand of car?
+          <select
+        id="cars"
+        name="cars"
+        multiple
+        >
+            <option value="BMW">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+        <label for="rate">How do you rate our work?
+          <input
+        type="range"
+        id="rate"
+        name="rate"
+        >
```

Comment:
```suggestion
          <input
            type="range"
            id="rate"
            name="rate"
          >
```
check your code to satisfy the [styleguide](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) 😉

Reply:
thanks. give me a minute